### ISSUE TITLE
replace Indiv species table calc code with SumRelAbund

### ISSUE THREAD

**Issue by aaronweed** [NCRN/BirdViz#75]
Replacing the code under #### Individual tables, titles, captions, basedata used to calculate other tables #### in server.R to use SumRelAbun would keep all analysis code in the package and provide options in the viz. Worth it?

**Comment by johnpauls** [comment]
Yes, whenever there is a function in the package that will simplify the visualizer, I am all in favor of making the switch. In the long run this will save on maintenance time mad make results more consistent.
